test/release/examples/primers/errorHandling.chpl

```diff
@@ -0,0 +1,143 @@
+// Error Handling
+//
+// This primer is a short tutorial on Chapel error handling. A detailed
+// outline can be found in the :ref:`readme-errorHandling` technote.
+//
+
+/*
+  .. _primers-errorHandling-handling:
+
+  Handling Errors
+  ---------------
+
+  Standard library functions that used to halt now throw errors for
+  developers to handle as they see fit. This is especially useful for
+  recovering from errors in :mod:`IO`.
+ */
+
+use IO;
+
+const f1 = "fake.txt";
+const f2 = "still_fake.txt";
+
+var f: file;
+
+  // halts on error
+  // f = try! open(f1, iomode.r);
+
+try {
```

Comment:
Let's sprinkle in some more comments here about what try/catch are doing.
E.g. here it could say, near/after the open call, that control flow goes to the catch blocks if an 'open' throws an error.